src/preact/index.js

```diff
@@ -69,6 +69,14 @@ export function createContext(value) {
   return preact.createContext(value, undefined);
 }
 
+/**
+ * @param {...PreactDef.Renderable} unusedChildren
+ * @return {!Array<PreactDef.Renderable>}
+ */
+export function toChildArray(unusedChildren) {
+  return preact.toChildArray.apply(undefined, arguments);
```

Comment:
so, how is this different from re-exporting? Or is it just for the the jsdoc typing?